server/celery_config.py

```
@@ -244,3 +244,55 @@
     logging.info("Registered %d custom tasks: %s", len(non_celery_tasks), non_celery_tasks)
+
+
+# ---------------------------------------------------------------------------
+# Worker pre-warming: initialize expensive singletons once at worker startup
+# so background chat tasks don't pay cold-start penalties on every invocation.
+# ---------------------------------------------------------------------------
+from celery.signals import worker_process_init
+
+
+@worker_process_init.connect
+def _prewarm_worker(**kwargs):
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `_prewarm_worker`

Add return type annotation: `None`

(ANN202)